lib/src/util/dart_type_utilities.dart

```diff
@@ -224,6 +224,16 @@ class DartTypeUtilities {
     }
     return false;
   }
+
+  static bool overridesMethod(MethodDeclaration node) => (node.parent
+          as ClassDeclaration)
+      .element
+      .allSupertypes
```

Comment:
I'm pretty sure you could do something like `new List().from(class.mixins)..addAll(interfaces)..addAll(supertypes)`.any((t) => t.lookup(m.name) != null)`

I'm not a dart performance expert, and don't know how efficiently `lookup` is implemented (or if that's even the right method/it works how you want here), but I suspect it would be faster in at least the negative case of where the method is not found and all inherited method/members must be inspected.

Reply:
Done. Thanks.